editor/doc_tools.cpp

```diff
@@ -1274,6 +1317,14 @@ Error DocTools::_load(Ref<XMLParser> parser) {
 								if (parser->has_attribute("is_bitfield")) {
 									constant2.is_bitfield = parser->get_attribute_value("is_bitfield").to_lower() == "true";
 								}
+								if (parser->has_attribute("is_deprecated")) {
+									String result = parser->get_attribute_value("is_deprecated");
+									constant2.is_deprecated = (result == "true" || result == "True" || result == "TRUE" || result == "1");
+								}
+								if (parser->has_attribute("is_experimental")) {
+									String result = parser->get_attribute_value("is_experimental");
+									constant2.is_experimental = (result == "true" || result == "True" || result == "TRUE" || result == "1");
+								}
```

Comment:
You could use `parser->get_attribute_value("is_experimental").to_lower() == "true"` like above.

The XML docs are not meant to be edited by hand so in theory we only need to support what we write. We don't have to support `1`.